integration-auth/src/main/java/org/symphonyoss/integration/authentication/jwt/JwtAuthentication.java

```diff
@@ -57,10 +84,20 @@
   private IntegrationBridge integrationBridge;
 
   @Autowired
+  private IntegrationProperties properties;
+
+  @Autowired
   private IntegrationHttpApiClient integrationHttpApiClient;
 
+  @Autowired
+  private RsaKeyUtils rsaKeyUtils;
+
   private IntegrationAuthApiClient apiClient;
 
+  private PublicKey podPublicSignatureVerifier;
+
+  private Date podPublicSignatureVerifierExpirationDate;
```

Comment:
If you want a unique value for all the application (only one public certificate on the cloud) you can define a constant value as the Guava cache key.